common/lib/transport/connectionmanager.js

```diff
@@ -66,6 +68,8 @@ var ConnectionManager = (function() {
 			params.echo = 'false';
 		if(this.format !== undefined)
 			params.format = this.format;
+		if(this.stream !== undefined)
```

Comment:
> It needs to be set as a connect params somewhere.

I was intending that it gets into `params` only because it's in `transportParams` and then, if it is still unspecified, the default for that transport is applied. But that disperses the logic into different places so in retrospect I'm fine with this.